src-packed/validator.js

```diff
@@ -62,12 +63,31 @@ export async function getMatches(text, matches) {
         console.log(result.error);
         return;
     }
-    result.sentences.forEach(sentence => {
+
+    for (var i = 0; i < result.sentences.length; i++) {
+        var sentence = result.sentences[i];
         if (sentence.sentiment !== "negative")
-            return;
+            continue;
         console.log(`Index: ${sentence.offset}, Negative sentiment: ${sentence.confidenceScores.negative}`)
         if (sentence.confidenceScores.negative <= 0.75)
-            return;
+            continue;
+
+        // Remove commonly used phrases with a negative sentiment
+        var regex = new RegExp(ignoreList.ignorablePhrases.join("|"), "gi");
```

Comment:
Perf question: can we join these up front?
```
export const ignorablePhrases =
    "error|" +
    "build failure|" +
    "test failure";
```
When the JavaScript is minified, it might merge all those `+` operators like a C# compiler would.

Reply:
Good idea, let me try to use the literal regex syntax as well and move it all into the new file.